Android/build.gradle

```diff
@@ -0,0 +1,255 @@
+apply plugin: 'com.android.library'
+apply plugin: 'kotlin-android'
+
+android {
+    compileSdkVersion 28
```

Comment:
I think we are using a higher version. also for the build tools. please check here: https://github.com/applicaster/zapp-platform-android 

Use the master branch as a reference.

Reply:
Done